packages/stat/tests/stat.test.tsx

```diff
@@ -32,7 +32,7 @@ describe("<StatGroup />", () => {
         </Stat>
       </StatGroup>,
     )
-    expect(findByTestId("group")).toBeTruthy()
```

Comment:
`findByTestId` is async - this seemed to be leaving async handlers open, which caused stuff to fail after the test had already passed (and the assertion would always pass regardless of the result, because the promise is always truthy).

For some reason my PR seems to have tweaked the order/performance of the tests slightly such that it would trigger reasonably reliably in CI (but only very intermittently locally)